### DISCUSSION TITLE
List of lists isn't updated dynamically

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#9093] [Other Feature Feedback, Questions, & Ideas] [answered]
# Actual Results

One tab has:
![image](https://user-images.githubusercontent.com/2119212/146681951-c70c6b31-eb5c-429d-bea2-c3a6d90ab2ae.png)

In another tab, I've added a third list:
![image](https://user-images.githubusercontent.com/2119212/146681959-5a34d901-0e3e-4f44-be3f-4a6301d6c4e4.png)

# Expected results
At the very least, upon adding a list in the current page, the list of lists at the top should be updated to show the newly created list.

In the also-ran category, it'd be kinda nice if all pages with lists were updated when the list of lists was changed.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the heads up! I'll flag this for the team — hopefully in the short term a refresh will do the trick 😊
